docs/server/php/_initialize.mdx

```diff
@@ -17,6 +23,9 @@ $this->statsig = new StatsigServer("server-sdk-key", $options);
 ### 🔥 Warning - You need to schedule a cron job 🔥
 
 :::caution
+#### V3.0+
+If you do not configure a cron job, SDK does not fire a network request to fetch the latest config value anymore, instead, config values fetched earlier will be used.
```

Comment:
wait there will be no values fetched earlier in this world right?  You MUST have a job to update the values

Reply:
If they setup the Localfile correctly, it should be fine